tests/components/notify/common.py

```diff
@@ -5,24 +5,21 @@
 """
 
 from homeassistant.components.notify import (
-    ATTR_DATA,
     ATTR_MESSAGE,
     ATTR_TITLE,
     DOMAIN,
     SERVICE_NOTIFY,
 )
+from homeassistant.core import HomeAssistant
 from homeassistant.loader import bind_hass
 
 
 @bind_hass
-def send_message(hass, message, title=None, data=None):
+def send_message(hass: HomeAssistant, message: str, title: str | None = None):
```

Comment:
This is an unrelated service helper for the notify service. We shouldn't change here.